### PR TITLE
Feature: New Pizza Trivia Added

### PR DESCRIPTION
**Pull request by mohdahsanrazakhan**
## Description
This PR adds the feature/add-pizza-trivia in **Pizza Trivia** list.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Related to #36

## Mobile & Desktop Screenshots/Recordings
Pizza Trivia

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
NO

## [optional] What gif best describes this PR or how it makes you feel?
PeachCatPizzaGIF

### CODE REVIEW

File: pizza-facts-&-trivia.md
Comment (adiati98) [inline pizza-facts-&-trivia.md:41]: Let's avoid wordy words here.

```suggestion
- How many *acres* worth of pizza do Americans eat daily?
  - 100 acres
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @mohdahsanrazakhan,
Thank you for your PR! 🙌

I left a review here for you to fix before we merge your PR.
Let me know when it's fixed and ready for a review.
Thank you. 🙂

**Comment by mohdahsanrazakhan** [comment]
Hi @adiati98 your commit suggestion is committed. Review it.

**Review by adiati98 — APPROVED** [review]
Thank you for the fix, @mohdahsanrazakhan!

For your future reference, it would be helpful for you if you pay more attention to your choice of words and not rush maintainers to review your PR. 🙂

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊

git merge GIF

**Pull request merged**
